gadget/validate.go

```diff
@@ -230,31 +240,73 @@ func ensureRolesConsistency(roles map[string]*roleInstance, expectedSeed bool) e
 	return nil
 }
 
+func ensureRolesConsistencyOnModalClassic(roles map[string]*roleInstance) error {
+	if roles[SystemBoot] == nil || roles[SystemData] == nil {
+		return fmt.Errorf("system-boot and system-data roles are needed on classic")
+	}
+
+	// Make sure labels are as expected - save is optional
+	roleLabelToCheck := []roleLabel{
+		{SystemBoot, ubuntuBootLabel}, {SystemData, ubuntuDataLabel}}
+	roleLabelOptional := []roleLabel{
+		{SystemSeed, ubuntuSeedLabel}, {SystemSave, ubuntuSaveLabel}}
```

Comment:
don't we have some of this defined below?

Reply:
Indeed, good catch